### PR TITLE
relabel MONDO:0017939 classic multiminicore myopathy

### LINKED ISSUE #6905: MONDO:0017939 minicore myopathy
**Mondo term (ID and Label):**
MONDO:0017939 minicore myopathy

**Suggested new label:**
Classic multiminicore disease

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**_Optional_: Any additional information (like supporting evidence, PubMed ID, etc.)**

"Minicore myopathy" is an exact match to this HPO term https://hpo.jax.org/app/browse/term/HP:0003789 and is causing some trouble for us in MedGen.  But both Orphanet and GARD use "Classic multiminicore myopathy" for their preferred names (and UMLS now has C5679883 for the OrphaNet record).

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6905

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
Minor changes so I went ahead and merged.
